packages/cashier/src/pages/account-transfer/account-transfer-locked/account-transfer-locked.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { useHistory } from 'react-router-dom';
 import { routes } from '@deriv/shared';
 import { Icon, Checklist, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
+import { TRootStore, TClientStore } from 'Types';
+
+type TAccountTransferLocked = {
```

Comment:
```suggestion
type TProps = {
```
Just a suggestion 🙇🏻

Reply:
This is a name convention we used across the cashier, T<ComponentName>Props.

However, I forgot to add Props at the end. nice notice. I will add them to their names.